dev_guide/compute_resources.adoc

```diff
@@ -1,25 +1,129 @@
-= Compute Resources
+= Quotas, Limit Ranges, and Compute Resources
 {product-author}
 {product-version}
 :data-uri:
 :icons:
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 == Overview
 
-Each container running on a node consumes compute resources, which are measurable quantities that can be requested, allocated, and consumed.
+Using link:#dev-quotas[quotas] and link:#dev-limit-ranges[limit ranges], cluster
+administrators can set constraints to limit the number of objects or amount of
+compute resources that are used in your project. This helps cluster
+administrators better manage and allocate resources across all projects, and
+ensure that no projects are using more than is appropriate for the cluster size.
 
-When authoring a pod configuration file, you can optionally specify how much CPU and memory (RAM) each container needs in order to better schedule pods in the cluster and ensure satisfactory performance.
+As a developer, you can also
+set link:#dev-compute-resources[requests and limits on compute resources] at the
+pod and container level.
 
-CPU is measured in units called millicores. Each node in a cluster inspects the operating system to determine the amount of CPU cores on the node, then multiplies that value by 1000 to express its total capacity. For example, if a node has 2 cores, the node's CPU capacity would be represented as 2000m. If you wanted to use 1/10 of a single core, it would be represented as 100m.
+The following sections help you understand how to check on your quota and limit
+range settings, what sorts of things they can constrain, and how you can request
```

Comment:
Maybe s/things/resources/ (unless "things" is more appropriate because the settings can affect API objects)?